src/x509.c

```diff
@@ -7202,6 +7231,433 @@ WOLFSSL_X509_CRL* wolfSSL_d2i_X509_CRL(WOLFSSL_X509_CRL** crl,
 
     return newcrl;
 }
+
+WOLFSSL_X509_NAME* wolfSSL_X509_CRL_get_issuer_name(WOLFSSL_X509_CRL* crl)
+{
+    WOLFSSL_ENTER("X509_CRL_get_issuer_name");
+    if (crl) {
+        return crl->crlList->issuer;
+    }
+    return NULL;
+}
+
+int wolfSSL_X509_CRL_version(WOLFSSL_X509_CRL* crl)
+{
+    if (crl->crlList == NULL)
+        return 0;
+    return crl->crlList->version;
+}
+
+int wolfSSL_X509_CRL_get_signature_type(WOLFSSL_X509_CRL* crl)
+{
+    if (crl->crlList == NULL)
+        return 0;
+    return crl->crlList->signatureOID;
+}
+
+int wolfSSL_X509_CRL_get_signature_nid(const WOLFSSL_X509_CRL* crl)
+{
+    if (crl == NULL)
+        return 0;
+
+    return oid2nid(crl->crlList->signatureOID, oidSigType);
+}
+
+int wolfSSL_X509_CRL_get_signature(WOLFSSL_X509_CRL* crl,
+    unsigned char* buf, int* bufSz)
+{
+    WOLFSSL_ENTER("wolfSSL_X509_CRL_get_signature");
+    if (crl->crlList == NULL || bufSz == NULL)
+        return WOLFSSL_FATAL_ERROR;
+
+    if (buf != NULL)
+        XMEMCPY(buf, crl->crlList->signature, *bufSz);
+    *bufSz = (int)crl->crlList->signatureSz;
+
+    return WOLFSSL_SUCCESS;
+}
+
+int wolfSSL_X509_REVOKED_get_serial_number(RevokedCert* rev,
+    byte* in, int* inOutSz)
+{
+    WOLFSSL_ENTER("wolfSSL_X509_REVOKED_get_serial_number");
+    if (rev == NULL || inOutSz == NULL) {
+        return BAD_FUNC_ARG;
+    }
+
+    if (in != NULL) {
+        if (*inOutSz < rev->serialSz) {
+            WOLFSSL_MSG("Serial buffer too small");
+            return BUFFER_E;
+        }
+        XMEMCPY(in, rev->serialNumber, rev->serialSz);
+    }
+    *inOutSz = rev->serialSz;
+
+    return WOLFSSL_SUCCESS;
+}
+
+/* human readable print out of CRL version
+* return WOLFSSL_SUCCESS on success
+*/
+static int X509CRLPrintVersion(WOLFSSL_BIO* bio, WOLFSSL_X509_CRL* crl,
+    int indent)
+{
+    int version;
+
+    if ((version = wolfSSL_X509_CRL_version(crl)) < 0) {
+        return WOLFSSL_FAILURE;
+    }
+
+    if (X509PrintVersion_ex(bio, version, indent) != WOLFSSL_SUCCESS) {
+        return WOLFSSL_FAILURE;
+    }
+
+    return WOLFSSL_SUCCESS;
+}
+
+/* print serial number out
+* return WOLFSSL_SUCCESS on success
+*/
+static int X509RevokedPrintSerial(WOLFSSL_BIO* bio, RevokedCert* rev,
+    int indent)
+{
+    unsigned char serial[32];
+    int  sz = sizeof(serial);
+
+    XMEMSET(serial, 0, sz);
+    if (wolfSSL_X509_REVOKED_get_serial_number(rev, serial, &sz)
```

Comment:
Does this print the hex all on a single line with no ':' delimiter? I.e `    Serial Number: 123456789`

Reply:
Fixed